### PR TITLE
Add Caesar Cipher module

### PR DESCRIPTION
**Pull request by ItzAtomikHD**

### CODE REVIEW

File: modules/caesarcipher.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+if (args.length < 1) {
+    ktaneSpeak("Not enough arguments.")
+    return;
+}
+
+var display = args[0];
+var offset = 0;
+var offsetLock = false;
+
+if (bombinfo.getPortsCount("parallel") > 0 && bombinfo.indicators.exists("nsa") && bombinfo.indicators.isLit("nsa")) {
```
Comment (mrmelon54) [inline modules/caesarcipher.js:10]: u can just do isLit coz if it is lit then it must exist

Comment (ItzAtomikHD) [reply modules/caesarcipher.js:10]: but what if it doesn't exist

Comment (mrmelon54) [reply modules/caesarcipher.js:10]: then isLit is false

Comment (ItzAtomikHD) [reply modules/caesarcipher.js:10]: oh okaay youc an edit it

**Pull request merged**